server/src/modules/stock/stock.service.ts

```diff
@@ -0,0 +1,58 @@
+import { PrismaClient } from "@prisma/client"
+
+const prisma = new PrismaClient()
+
+export const createStockService = async ({
+  quantity,
+  price,
+  purchasePrice,
+  productId,
+}: {
+  quantity: number
+  price: number
+  purchasePrice: number
+  productId: string
+}) => {
+  return await prisma.stock.create({
+    data: { quantity, price, purchasePrice, productId },
+  })
+}
+
+export const getStockService = async () => {
+  return await prisma.stock.findMany({})
+}
+
+export const updateStockService = async ({
+  id,
+  quantity,
+  price,
+  purchasePrice,
+  productId,
+}: {
+  id: string
+  quantity: number
+  price: number
+  purchasePrice: number
+  productId: string
+}) => {
+  return await prisma.stock.update({
+    where: {
+      id,
+    },
+    data: {
+      quantity,
+      price,
+      purchasePrice,
+      productId,
+    },
+  })
+}
+
+export const deleteStockService = async ({ id }: { id: string }) => {
+  const data = await prisma.stock.delete({
```

Comment:
We can return directly here like `return await prisma.stock.delete({})` instead of defining `data` variable and returning later.

Reply:
done